BrainPortal/app/controllers/userfiles_controller.rb

```diff
@@ -1333,16 +1333,19 @@ def export_file_list
     # Create the new file list
     file_list = CbrainFileList.new(
       :user_id          => current_user.id,
-      :group_id         => current_project.try(:id) || current_user.own_group.id,
+      :group_id         => if current_project.try(:public) then current_user.own_group.id else current_assignable_group.id end,
```

Comment:
Please don't use a if then else, it's really ugly. if then else are more appropriate for statements, not for expressions, even though they work as expressions.

```ruby
:group_id         => (current_project.try(:public) ? current_user.own_group.id : current_assignable_group.id),
```

Reply:
ok